### DISCUSSION TITLE
Is tun2socks compatible with Tap ?

### DISCUSSION THREAD

**Discussion by chromer030** [xjasonlyu/tun2socks#200] [General]
Currently `tun` interfaces are well supported and working , is it possible for `Tap` support ?

**Comment by xjasonlyu** [comment]
No, wintun only supports layer 3 tun driver. And why would you bother to use TAP anyway?

**Reply by chromer030** [reply]
Everywhere i see [tun/tap](https://en.wikipedia.org/wiki/TUN/TAP) together , it was a question for me , can we use tap interfaces too , of course i know all differences between tun and tap (layers etc).
The tool is working well with tun, Thanks for this great tool.
